File: 0005-Day 5 - Introduction to Problem Solving/Q4. Pattern Printing -1/Solution.cs
class Solution {
    public List<List<int>> solve(int A) {

        List<List<int>>arr=new List<List<int>>();
        
        for(int i=0;i<A;i++){
            List<int>ar=new List<int>();
            for(int j=0;j<A;j++){
               if(j<=i){
                    ar.Add(j+1);
               }
               else{
                    ar.Add(0);
               }
            }
            arr.Add(ar);
        }

        return arr;
    }
}

File: 0005-Day 5 - Introduction to Problem Solving/Q4. Pattern Printing -1/Solution.java
public class Solution {
    public int[][] solve(int A) {

        int[][]arr=new int[A][A];
        for(int i=0;i<A;i++){
            for(int j=0;j<=i;j++){
                arr[i][j]=j+1;
            }
        }

        return arr;
    }
}
